addons/ace_tweaks/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -0,0 +1,21 @@
+
+class ace_medical_treatment_actions {
+
+    // - Syringes -------------------------------------------------------------
+    class Morphine;
+    class Painkillers: Morphine {
```

Comment:
GVAR(Painkillers)

Reply:
This class is part of the unfinished ace implementation